vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -290,13 +290,11 @@ func (e *StandardTxExecutor) AddValidatorTx(tx *txs.AddValidatorTx) error {
 		return err
 	}
 
-	txID := e.Tx.ID()
-	newStaker, err := state.NewPendingStaker(txID, tx)
-	if err != nil {
+	if err := e.addStakerFromStakerTx(tx); err != nil {
```

Comment:
scaffolding, but it's going to be very helpful as we handle non-scheduled stakers in later PRs